_Sidebar.md

```diff
@@ -9,6 +9,7 @@
     * [Building for mobile](./Building-for-mobile)
 * [Configuring Client Libs](./Configuring-Client-Libs)
 * [Mock vs. non-mock](./Mock-vs.-non-mock)
+    * [Mock implementation](./Mock-Implementation)
```

Comment:
One final thing: can we change the name of the file to "Mock implementation"? Minor point but it's the convention we've been following (for simplicity and consistency).

Reply:
("implementation" in lowercase)